bin/cmd.js

```diff
@@ -16,11 +16,11 @@ var regexTester = require('safe-regex-test');
 var jsFile = regexTester(/\.js$/i);
 
 var argv = minimist(process.argv.slice(2));
-var tap = faucet({
-	width: defined(argv.w, argv.width, process.stdout.isTTY
-		? process.stdout.columns - 5
-		: 0)
-});
+var opts = {
+	width: defined(argv.w, argv.width, process.stdout.isTTY ? process.stdout.columns - 5 : 0),
+	stack: defined(argv.stack, process.env.SHOW_STACK ? process.env.SHOW_STACK === 'true' : true)
```

Comment:
generally it's best to have absence and `false` be the same - can we invert this to "hideStack" instead?

(altho i do like `--no-stack` as the CLI arg name)

Reply:
Thanks